### PR TITLE
DM-33922: exposurelog and narrativelog: fix value namespace

### PR DESCRIPTION
**Pull request by r-owen**
Moving the charts from charts to this package broke value namespaces.
Rather than move the some of the names up a level, I put
application-specific values into namespace "config".

I also removed some values from values.yaml, replacing them with
comments, so that the site-specific file MUST override some values
in order for deployment to work. This is to avoid the current situation
where the service may run with incorrect default values.

### CODE REVIEW

**Review by afausti — CHANGES_REQUESTED** [review]
That seems good. Because you moved your charts to Phalanx and they essentially became wrapper charts all your configuration values must be at the top level in the `values.yaml` and `values-*.yaml` files.
You also need to list `values.yaml` with the default configuration and the `values-*.yaml` with the environment-specific configuration in the Argo CD application definition for `exposurelog` and and `narrativelog`. You can find those files under `phalanx/science-platform/templates`.

File: services/exposurelog/Chart.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.2
+version: 0.3.3
```
[the remark this answers is not in the document]
Comment (r-owen) [reply services/exposurelog/Chart.yaml]: I'm very glad to hear that. Bumping this version number felt like a waste of time.

**Comment by r-owen** [comment]
I have made all your suggested changes (I think) and tested the results at the base. Both narrativelog and exposurelog are now deployed correctly there. The value for site_id is correct for both, and the values for butler_uri_1 and butler_uri_2 are correct for exposurelog (narrativelog has no such values). This doesn't test all values, but it's the best I can do without deploying on the summit (which uses a few more variables).

**Comment by r-owen** [comment]
And I just rebased to master.

**Comment by r-owen** [comment]
Good thing I checked the summit deployment. The big domain name rename they are doing required changes to both nfs_server_x values. Updated and waiting for things to rebuild so I can test the new values.

**Review by afausti — APPROVED** [review]
Nice, the changes look good, glad everything is working.

**Pull request merged**